clients/ui/frontend/src/app/pages/modelCatalog/screens/RegisterCatalogModelForm.tsx

```diff
@@ -195,6 +197,14 @@ const RegisterCatalogModelForm: React.FC<RegisterCatalogModelFormProps> = ({
                 title="Additional model metadata, such as labels, provider, and license, will be available to view and edit after registration is complete."
               />
             </StackItem>
+            {artifactsLoadError && (
+              <StackItem>
+                <Alert variant="warning" isInline title="Unable to load model artifacts">
+                  {artifactsLoadError.message}. The model location field will be empty and must be
+                  filled manually.
```

Comment:
not sure about the error message - we can remove it

Reply:
I think we don't need this here. As we discussed here https://github.com/kubeflow/model-registry/pull/1615#issuecomment-3298705519 
if we don't have uri - we are disabling the register button itself. So I think we will not hit this case may be?